docs/mint.json

```diff
@@ -371,23 +371,23 @@
     },
     {
       "source": "/apps/usecases/langchain_blogs/:slug2*",
-      "destination": "https://composio.dev/tools/:slug2*"
+      "destination": "https://app.composio.dev/apps/:slug2*"
```

Comment:
Consider adding version tracking to help monitor URL structure changes. This can be added at the root level of the configuration:

```json
{
  "version": "2.0.0",  // Track major URL structure changes
  "urlSchema": "app.composio.dev/apps",  // Document current URL schema
  // ... rest of the configuration
}
```